### PR TITLE
Add healthiness report dropdown to dashboard summary tab

### PR DESCRIPTION
**Pull request by bryan-lou**

### CODE REVIEW

File: web/src/testgrid_healthiness-summary.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { LitElement, html, css } from 'lit';
```
Comment (chases2) [inline web/src/testgrid_healthiness-summary.ts:1]: The name of this file should be all hyphens, I think, to match the others

**Review by chases2 — COMMENTED** [review]
Do you have a screenshot or test to show what the healthiness report looks like?

**Review by sultan-duisenbay — CHANGES_REQUESTED** [review]
Thanks for PR, Bryan! Please address Sean's comments and resolve the merge conflicts from your previous PR.

**Comment by bryan-lou** [comment]
> Do you have a screenshot or test to show what the healthiness report looks like?

Hi @chases2 , here are few screenshots which show what they look like in the dashboard summary
Screenshot 2023-06-20 at 3 04 23 PM
Screenshot 2023-06-20 at 3 04 39 PM
Screenshot 2023-06-20 at 3 08 19 PM

File: web/src/testgrid_healthiness-summary.ts (hunk 1 above)
Comment (bryan-lou) [reply web/src/testgrid_healthiness-summary.ts]: Done

**Comment by chases2** [comment]
Looks good! Would love to see some simple Jasmine tests that assert that data in the structure is displayed in the component. You may also consider creating a storybook entry for this component in a follow-up PR.

/ok-to-test
/lgtm

**Review by sultan-duisenbay — APPROVED** [review]
LGTM! Please proceed with tests and stories

**Pull request merged**